docs/img/UnsubscriptionSequence.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2020-08-18T11:03:49.043Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.125 Safari/537.36" etag="wrSFe2ZkkFSsJns1IMVC" version="13.4.9" type="device"><diagram id="kgpKYQtTHZ0yAKxKKP6v" name="Page-1">5VfbbqMwEP0apO5DIzAlSR9D05vUKtGmq7b7ZsABKwYjY5qwX7+2MbdA2qbdSrvaKlI9x2ODz5wZD4Z9Ee+uGUyjexogYgAz2Bn23ADAOgPAkD8zKEpkYtklEDIcaKcGWOFfSIOmRnMcoKzjyCklHKdd0KdJgnzewSBjdNt1W1PSfWoKQ9QDVj4kffQRBzwq0aljNvgNwmFUPdky9UwMK2cNZBEM6LYF2ZeGfcEo5eUo3l0gIsmreCnXXR2YrV+MoYS/Z4Gd5O7Vjp37xZ2ZJovHBwCvT/UuL5Dk+sCzNDXA2AC2iIDtnmS5l/kMe4h908fgRcWNOFEqh3lM7vAaEZwIy00RwzHiiIkZouFlg7kiUhwKTM5byiYEphn21LbyoQz5OcvwC/qOslIQCqV5EqBAr6rZlFMZZ3RTx0dO98mpTooYR7sWpMm6RlS8ICuESzU71oHTyp1qc9vIoA521JKAXYFQSy+st26iIwY6QEcEC/SC1QtJSnHC1XMd13Dme+GgjEc0pAkk7YA0xJofIfZVXb2b7TOnQ/ZkiOw+19b5V3Ht9Li+yb3/LAPss78tA8b/TgY4x7K9lwGDZA9kwPiruJ72uXbc5WL1IMAfSX0vSI73YyAPjcUdOiM4TATkUc5pLLhDSTCTl7LECPU3ChIv+ySAU3MEKvtZsj8ynbejgXaYy9XWqLLKtZU13+mlyihaRksAJSYf3Lgrq2hb+wsOZlhGc+ajtws5hyxEr2lIyx0FnR6lryCGCOSiWnS7nwFB6KVLmSOtPLe60utpqjyQXrUnq/o1Pq608yGlAVNiM99HKRep+UmVvaGitajIuve0pB3ALKoLba1QqytPMHb+qGy+PMzTbpiBORnZjtn8gXdFvbcvsI7bt5T9K/tWjnS9ztBnFRfcPpvk6ae7uvWzeeTfL2fn24HaptpeM2/qWmbIfnpMRIxcj4lRKEecyvqbeyPxTzUEIigxfVHeUG2xZkKDQqfVPkE5oxwSafiEZsqfR0jWd+SpkuFv5FHNE7xWnsXhdjuhXHYZ2whztEqhqjRb8fUlsIjHRAvVg/4mVBfbIudlYzIvpc02C7EK86Ipst3OAuqEImgtz6w7EGtSi7d3qQ3o+eA951jdrsIauunAwFU3Pf6mE2bzgVWKp/lMtS9/Aw==</diagram></mxfile>
```

Comment:
The swim lanes on this diagram are pretty long considering there are only two interactions. Could they be shortened?

Reply:
Yes, will do